modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/Replicator.java

```diff
@@ -1535,6 +1541,7 @@ void sendEntries() {
      */
     private boolean sendEntries(final long nextSendingIndex) {
         final AppendEntriesRequestBuilder rb = raftOptions.getRaftMessagesFactory().appendEntriesRequest();
+        rb.timestamp(options.getNode().clockNow());
```

Comment:
We have agreed to send timestamps only in heartbeats

Reply:
Yes, there is no need to use a clock here. Fixed.